app/views/accounts/registrations/new.html.erb

```diff
@@ -1,45 +1,63 @@
+  <ul>
+    <li>Compete <u>officially</u> in the NZIC (New Zealand Informatics Contest).</li>
+    <li>Potentally be invited to a programming camp run by the NZOI (New Zealand Olympiad in Informatics)</li>
+  </ul>
+  <p>Also note that:</p>
+  <ul>
+    <li>Anyone is welcome to create an account and use our site without specifying a graduation date.</li>
+    <li>Anyone can compete in the NZIC unofficially without specifying a graduation date. However, you will not be ranked on offical scoreboards or be eligible to be invited to our programming camps.</li>
+    <li>Only New Zealand citizens or perminant residents, with a valid future graduation date, will be considered for our international IOI team.</li>
+  </ul>
+  <p><b>Please select approximate secondary school graduation date or leave unspecified:</b></p>
   <%= f.fields_for :school_graduation do |graduation_form| %>
-  <%= graduation_form.radio_button('enabled', false, checked: !@user.school_graduation) %><%= f.label(:school_graduation_enabled_false, "Unspecified (Not considered eligible for IOI)") %> <br />
+  <%= graduation_form.radio_button('enabled', false, checked: !@user.school_graduation) %><%= f.label(:school_graduation_enabled_false, "Unspecified") %> <br />
   <%= graduation_form.radio_button('enabled', true, checked: !!@user.school_graduation) %><%= select_month(@user.school_graduation || 12, {prefix: 'user[school_graduation]'}, class: :radio_input) %><%= select_year(@user.school_graduation || Date.today, {start_year: [Date.today.year-50,@user.school_graduation.try(:year) || Date.today.year].min, end_year: Date.today.year+18, prefix: 'user[school_graduation]'}, class: :radio_input) %>
-  <% end %></p>
+  <% end %>
 
-  <p><%= f.label :school %><br />
+  <h3><%= f.label :school %></h3>
   <%= f.fields_for :school, @user.school do |school_form| %>
     <%= school_form.text_field :name, class: [:school_select, :jsnodisplay] %>
     <%= f.select :school_id, School.where(country_code: "NZ").order(:name).collect {|s| [s.name, s.id]}, { include_blank: true }, class: [:school_select, :jsdisplay], disabled: true %>, 
     <%= school_form.country_select(:country_code, only: ["NZ"] ) %><br />
     <a class="school_select togglelink jsnodisplay" data-toggle="school_select" href="javascript:void(0)">Select school from list</a>
-    <a class="school_select togglelink jsdisplay" data-toggle="school_select" href="javascript:void(0)">Add new school</a>
+    <a class="school_select togglelink jsdisplay" data-toggle="school_select" style="color:blue" href="javascript:void(0)">Add new school</a>
   <% end %>
   </p>
 
   <p>
     <%= recaptcha_tags :site_key => @db_settings["recaptcha/public_key"] if @db_settings["recaptcha/public_key"] && !@db_settings["recaptcha/public_key"].empty? %>
   </p>
+  <h3>Submit</h3>
```

Comment:
Move the reCAPTCHA below "Submit" instead of "School".
```suggestion
  <h3>Submit</h3>
  <p>
    <%= recaptcha_tags :site_key => @db_settings["recaptcha/public_key"] if @db_settings["recaptcha/public_key"] && !@db_settings["recaptcha/public_key"].empty? %>
  </p>
```

Reply:
Good spot